src/scrapers/base-isracard-amex.ts

```diff
@@ -98,6 +116,39 @@ interface ScrapedTransactionData {
   }>;
 }
 
+function getCreditUtilizationUrl(servicesUrl: string) {
+  return buildUrl(servicesUrl, {
+    queryParams: {
+      reqName: 'RikuzNetuneyCreditDigi',
+    },
+  });
+}
+
+async function fetchCreditUtilization(
+  page: Page,
+  servicesUrl: string,
+): Promise<_.Dictionary<AccountCredit> | null> {
```

Comment:
Please use standard types, I'm not familiar with `lodash` types, I can't understand them from the tooltip and I don't expect other contributors to understand it.

Reply:
As lodash is already installed on the project I believed it's OK.
Will change

Reply:
You're right, although I'm thinking a lot before I add `lodash` to a project, if it is already there I accept using it.

But still, for the function signature, which is a kind of contract/interface, I prefer to use standard types.